src/app/components/CommentsList/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Text, View, TouchableOpacity } from 'react-native';
+import { Book } from '@interfaces/book';
+
+import { DATA } from './constants';
+import styles from './styles';
+import CommentCell from './Comment';
+
+interface Props {
+  book: Book;
+}
+
+// will need the book when I fetch the comments from the repository
```

Comment:
Add `TODO` comment: `// TODO: will need the book when I fetch the comments from the repository`